repos/spack_repo/builtin/packages/nvshmem/package.py

```diff
@@ -42,22 +52,83 @@ class Nvshmem(MakefilePackage, CudaPackage):
         when="@2.6:",
         description="Build with support for GPU initiated communication",
     )
+    variant("libfabric", default=False, description="Build with Libfabric support")
```

Comment:
Do you know if `+libfabric` conflicts with `+ucx` or vice versa?

Reply:
AFAICT, they are completely independent.